wsimod/nodes/nodes.py

```diff
@@ -793,7 +793,25 @@ def __init__(self, capacity=0, area=1, datum=10, initial_storage=0):
         else:
             self.storage = self.empty_vqip()
             self.storage_ = self.empty_vqip()  # Lagged storage for mass balance
+    
+    def apply_overrides(self, overrides: Dict[str, Any] = {}):
+        """Apply overrides to the tank.
 
+        Enables a user to override any of the following parameters: 
+        area, capacity, datum.
+        
+        Args:
+            overrides (dict, optional): Dictionary of overrides. Defaults to {}.
+        """
+        self.capacity = overrides.pop("capacity", 
+                                      self.capacity)
+        self.area = overrides.pop("area", 
+                                  self.area)
```

Comment:
Should this also recalculate `depth`?

```suggestion
                                  self.area)
                self.depth = self.capacity / self.area
```

Reply:
Unfortunately, there is no `depth` for `Tank` or its superclasses. I had a scan and find only `River` has `current_depth` in all the superclasses of `Storage`. `depth` is introducted in `Surface` and inherited by its superclasses.